.github/workflows/ci.yml

```diff
@@ -764,35 +764,40 @@ jobs:
       matrix:
         target:
           - platform: android
-            architecture: arm64-v8a
+            arch: armv8.2-a
```

Comment:
This is renamed to armv8.2-a to sync with the decision #11423. I might have some better ideas around these architecture names but I'll need to study a little more. In the meantime, I think it's fine to use these terms for now.